static/linux/webtorrent-desktop.ejs

```diff
@@ -5,7 +5,7 @@ Name=<%= productName %>
 <% if (genericName) { %>GenericName=<%= genericName %><% } %>
 <% if (description) { %>Comment=<%= description %><% } %>
 Icon=<%= name %>
-<% if (name) { %>Exec=<%= name %> %U<% } %>
+<% if (name) { %>Exec=<%= name %> --no-sandbox %U<% } %>
```

Comment:
Can you add a comment after each line with `--no-sandbox` to note that this is a temporary hack? Something like:

```
<!-- HACK: --no-sandbox fixes an Electron 6 bug. See: #1703 -->
```

Reply:
Sure! 👍